packages/core/test/stylable-transformer/at-property.spec.ts

```diff
@@ -1,6 +1,7 @@
+import { processorWarnings } from '@stylable/core';
+import { expectWarningsFromTransform, generateStylableResult } from '@stylable/core-test-kit';
 import { expect } from 'chai';
 import type * as postcss from 'postcss';
-import { generateStylableResult } from '@stylable/core-test-kit';
 
 describe('@property support', () => {
```

Comment:
I think all these tests should move to the `css-vars.spec` file, not sure about this as a separate scope.

Reply:
Let's discuss it in a separate issue.